src/home/swap-container/components/swap/index.tsx

```diff
@@ -694,6 +703,29 @@ const Swap = ({
     </StyledButton>
   );
 
+  const minimumTokensNeeded = usdPriceToToken(
+    from,
+    MINIMUM_USD_RATE_FOR_DEPOSIT[currentNetwork.chainId] || DEFAULT_MINIMUM_USD_RATE_FOR_DEPOSIT,
+    usdPrice
+  );
+
+  const NoMinForDepositButton = (
+    <StyledButton size="large" color="default" variant="contained" fullWidth disabled>
+      <Typography variant="body1">
+        <FormattedMessage
+          description="disabledDepositByUsdValue"
+          // eslint-disable-next-line no-template-curly-in-string
+          defaultMessage="You have to invest at least a rate of ${minimum} USD ({minToken} {symbol}) to enable this option."
```

Comment:
For this copy, I suggest: `You have to invest at least a rate of ${minimum} USD ({minToken} {symbol}) per day to create a position`